robot/basestation/static/js/app.js

```diff
@@ -10,8 +10,8 @@ $(document).ready(() => {
         $('#serial-type').text(getCookie('serialType'))
       }
 
-      // only for ARM and ROVER (for now)
-      if (window.location.pathname != '/science') {
+      // only for ARM, ROVER and SCIENCE (for now)
+      if (window.location.pathname != '/pds') {
```

Comment:
That's why it's here. If the route is _not_ pds, then add click event listeners for camera stream buttons.

Reply:
This `if` is completely useless. If jQuery does not find `$('#front-camera-stream-btn')`. It will just do nothing. This mean it is safe to just remove the if and avoid having to modify it every time we add a new page that doesn't have the cameras. Ideally the javascript related to cameras should only run when a camera related template is added, but that is issue  #141

So for now I would just remove the if so we don't have to change it every now and then.

Reply:
Although to be fair an improvement would be to check if the id "front-camera-stream" exists instead of checking the URL. This would reduce the maintenance